### ISSUE TITLE
why delta times the norm of direction when compute the weights of samples?

### ISSUE THREAD

**Issue by AAAeray** [google-research/multinerf#125]
Hi, 
Thanks for your great work, I do not understand this:
`def compute_alpha_weights(density, tdist, dirs, opaque_background=False):
  """Helper function for computing alpha compositing weights."""
  t_delta = tdist[..., 1:] - tdist[..., :-1]
  delta = t_delta * jnp.linalg.norm(dirs[..., None, :], axis=-1)
  density_delta = density * delta`
why delta times the norm of direction?
